docs/home/release-notes/v1.50-v1.59.md

```diff
@@ -24,6 +24,26 @@ Things that you have keep in mind before upgrading process:
 
 - Optional: We've developed [a tool that checks consistency of data](../troubleshooting/sanity-check.md), you could use it before and after the upgrading process and compare results of those checks just to be sure that everything went as expected
 
+#### Following steps
+
+  1. Install the [odm-ops](../helm/how-to-deploy.md#deployment-process) helm chart, it will install the Altinity ClickHouse operator with pre-configured settings.
+
+  2. Copy the parameters **you use** from ClickHouse to Altinity ClickHouse (requests, limits, disk size +25%, etc.). You can refer to the example `migrate-clickhouse-to-clickhouse-cluster.yaml`.
```

Comment:
I don't see that we need a separate example for this.

I'd prefer to go with safer recommendations and leave only disk size.

Reply:
Some clients use limits and requests and I see the point in specifying them too, so that they understand where and how to transfer parameters